src/libprojectM/Renderer/Shape.cpp

```diff
@@ -177,7 +177,9 @@ void Shape::Draw(RenderContext& context)
                            glm::value_ptr(context.mat_ortho));
         glVertexAttrib4f(1, border_r, border_g, border_b, border_a * masterAlpha);
         glLineWidth(1);
+#if USE_GLES == 0
```

Comment:
I thought that we used `#ifndef` other places.. should double-check that

Reply:
Yeah, we did, and it caused issues as [it's a 0/1 define](https://github.com/projectM-visualizer/projectm/blob/460a2dcd840c7651bcf22f877793178983841d20/config.h.cmake.in#L67), so the `#ifdef` was always true'ish. Some other defines are simple #define/#undef, but not this one.